package.json

```diff
@@ -560,6 +559,7 @@
     "chalk": "^4.1.2",
     "chokidar": "^3.6.0",
     "concurrently": "^8.2.2",
+    "contentful": "^10.8.7",
```

Comment:
I have moved this from "dependencies" to "dev-dependencies". We only use this package for its types, so we don't strictly need this in our builds.

Reply:
As this is now a dev dependency, it also resolves the `yarn audit` [issues surrounding axios](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/95274/workflows/b92333ec-1a1c-4ce6-bd7d-7601fe14e1e3/jobs/3546366).